lib/tasks/services.rake

```diff
@@ -12,4 +12,17 @@ namespace :services do
   task :destroy_marked_as_deleted => :environment do
     DestroyAllDeletedObjectsWorker.perform_later(Service.to_s)
   end
+
+  desc 'Destroy a service in the Background as long as the provider has at least 1 more service. Whether it is a default one or not'
+  task :destroy_default_service, %i[account_id service_id] => [:environment] do |_task, args|
+    account = Account.providers.find(args[:account_id])
+    service_id = args[:service_id]
+    service = account.services.find(service_id) # So it will raise an error if this service does not belong to this account :)
+    if service.default?
+      new_default_service = account.services.where.not(id: service_id).first! # It will raise an error if it does not have another service
+      account.update!({default_service_id: nil}, without_protection: true)
+      ThreeScale::Core::Service.make_default(new_default_service.backend_id)
```

Comment:
https://github.com/3scale/porta/pull/1886/files#r428142874

I think these 2 lines should be abstracted to
```ruby
new_default_service.make_default_backend_service
```